src/main/java/frc/robot/SwerveConstants.java

```diff
@@ -0,0 +1,82 @@
+package frc.robot;
+
+import edu.wpi.first.math.geometry.Rotation2d;
+import frc.lib.util.swerveUtil.RevSwerveModuleConstants;
+
+
+/**
+ * This file comes with command robot projects, and is intended to contain
+ * configuration information.
+ * I think it would be prudent if this file only contained CanIDs, because it
+ * is useful to have all the ids for the whole robot in one place.
+ * other configuration goes into subsystem specific configuration files,
+ * to make sure this one isn't cluttered.
+ */
+public final class SwerveConstants 
+{
+    public static final double stickDeadband = 0.1;
+    public static final double limelightOffset = 3;
+  
+
+    public static final class REV
+    {
+        public static final int pigeonID = 41;
+
+       
+    
+    }
+    public static final class Swerve {
+        /* Module Specific Constants */
+    /* Front Left Module */
+        public static final class Mod0 { 
+
+            public static final int driveMotorID = 11;
+            public static final int angleMotorID = 21;
+            public static final int canCoderID = 31;
+            // 0.046143 + 0.5 for offset
+            public static final Rotation2d angleOffset = Rotation2d.fromDegrees(-0.473389 - 13.45 + 15.56 + 12.39); //
```

Comment:
Magic numbers.  Where are they from?

Reply:
We found these offsets from PhoenixTuner, and the added pluses and minuses are from publishing our encoder values to SmartDashboard and adjusting as necessary. I could do the math and find a singular number for the offset, but I didn't want to mess anything up as it works right now as is. I asked the veterans and they said this is alright as well - however, if you disagree, I can fix it. Nevertheless, I'll add comments explaining my process for finding these magic numbers.